src/components/Controls.vue

```diff
@@ -31,10 +31,14 @@ export default {
         links: []
       },
       nodeMarkerSize: 50,
+      nodeMarkerType: "Circle",
```

Comment:
Just an idea for the future: rather than a string, you might want to consider using [Symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) to implement a small enum system for the different `nodeMarkerType` values.

Reply:
Agreed. I'll keep this in mind for future updates